mcs/class/System/System.Security.Cryptography.X509Certificates/X509BasicConstraintsExtension.cs

```diff
@@ -1,236 +0,0 @@
-//
```

Comment:
We cannot replace these extension classes with the CoreFX version until we have the full PAL support.